src/sysl/reljam/reljam.py

```diff
@@ -212,3 +213,8 @@ def main():
     if entities:
         raise RuntimeError('Some entities not output as expected: ' +
                            ', '.join(sorted(entities)))
+
+
+if __name__ == '__main__':
+    main()
+    sys.exit(0)
```

Comment:
Redundant

Reply:
Actually, good point.  The `if __name__ == '__main__':` entry point is in `reljam/__main__.py`.  When you run `reljam` it with `python -m` only the package needs to be specified e.g. `python -m sysl.reljam`

Reply:
Is was referring only to the sys.exit call. I don't know about the surrounding context.

Reply:
Ok.
See my comment as an additional one then.